5-Heroku/server/db/getAllItemsFromDB.js

```
@@ -0,0 +1,11 @@
+const itemModel = require('../models/item')
+
+module.exports = async function getAllItemsFromDB(userID) {
+    const query = itemModel.find({ userID: userID })
+    try {
+        return await query.exec()
+    } catch (err) { 
+        console.log(err)
+        return {} 
```

u want to throw the error back to who ever called u b/c u want them to know that somethign went wrong. it's ok to log it like u did but instead of return do `throw err`
and u need to handle it down the call stack